src/containers/blocks.jsx

```diff
@@ -124,7 +126,7 @@ class Blocks extends React.Component {
         const varListButtonCallback = type =>
```

Comment:
I'm kinda surprised these things are happening here. Do we want to move them into the scratch-blocks inject at some point? Or is there a reason they need to be controlled from the gui?

Reply:
Which things specifically? I agree that some of this may/should be relocatable, maybe we should file a general issue for refactoring this file to move stuff into scratch-blocks?